http/mixed-content.json

```
@@ -0,0 +1,320 @@
+{
+  "http": {
+    "mixed_content": {
```

Either you use `mixed-content` for both the ID and file name, or `mixed_content` for both. You can't mix these (see the linter failure),

```suggestion
    "mixed-content": {
```

---

Thanks. I'll change this one so it matches data-urls